examples/mach_eval_examples/bspm_eval/bspm_evaluator.py

```diff
@@ -1,8 +1,12 @@
 import os
 import sys
 
+
+os.chdir(os.path.dirname(__file__))
+
 # add the directory 3 levels above this file's directory to path for module import
 sys.path.append(os.path.dirname(__file__)+"/../../..")
+sys.path.append(os.path.dirname(__file__)+"/../../../..")
 sys.path.append(os.path.dirname(__file__))
```

Comment:
Why are you changing these lines when you are not changing the imports? Does the code not work as written in `develop`?

Reply:
Yeah, the code in `develop` did not run in my PC. One reason was because my working directory was different before running the code, so I added line 5. I also had to add line 9 because I was getting import issue in the analyzer in [line 7](https://github.com/Severson-Group/eMach/blob/ab4ab303f55a7831e5e2813ce067d0e3e2ebd144/mach_eval/analyzers/electromagnetic/bspm/jmag_2d.py#L7C1-L9C2).